common/src/main/scala/com/gu/media/model/MediaAtom.scala

```diff
@@ -123,7 +123,7 @@ case class MediaAtomBeforeCreation(
 
   // when creating an atom, use the `title` (headline) and `description` (standfirst) as the initial `youtubeTitle` and `youtubeDescription`
   override val youtubeTitle: String = title
-  override val youtubeDescription: Option[String] = description
+  override val youtubeDescription: Option[String] = YoutubeDescription.clean(description)
```

Comment:
this is the main change: clean `description` (aka standfirst) before setting it as `youtubeDescription` on atom creation